source/grid/grid_tools.cc

```diff
@@ -4191,14 +4191,52 @@ namespace GridTools
     const std::vector<Point<spacedim>> &points,
     const typename Triangulation<dim, spacedim>::active_cell_iterator
       &cell_hint)
+  {
+    const auto cqmp = compute_point_locations_try_all(cache, points, cell_hint);
+    // Splitting the tuple's components
+    const auto &cells    = std::get<0>(cqmp);
+    const auto &qpoints  = std::get<1>(cqmp);
+    const auto &maps     = std::get<2>(cqmp);
+    const auto &n_points = std::get<3>(cqmp);
+    // If a point was not found, throwing an error, as the old
+    // implementation of compute_point_locations would have done
+    AssertThrow(std::get<3>(cqmp).size() == 0,
+                ExcPointNotFound<spacedim>(points[n_points[0]]));
```

Comment:
you'll have to do `(void)n_points;` somewhere here to avoid the warning about an unused variable.

Alternatively, just inline the initializer of the variable into this one location here.